tcr-engine/events/tcr_event_test_data_builder.go

```diff
@@ -79,10 +79,38 @@ func WithModifiedTestLines(count int) func(filter *TcrEvent) {
 	}
 }
 
-// WithAddedTestCases sets added test cases to TCR event test data builder
-func WithAddedTestCases(count int) func(filter *TcrEvent) {
+// WithTotalTestsRan sets the total added test cases to TCR event test data builder
```

Comment:
Comment should be something like: 
// WithTotalTestsRun sets the total number of test cases run to TCR event test data builder